src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -86,24 +83,18 @@ export default class HexagonLayer extends Layer {
     }
   }
 
-  getPickingInfo(opts) {
-    const info = super.getPickingInfo(opts);
-    const pickedCell = this.state.pickedCell;
+  _onPickSubLayer(handler, info) {
+    const pickedCell = info.picked && info.index > -1 ?
+      this.state.hexagons[info.index] : null;
```

Comment:
It's reasonable for the user to get some aggregated information (e.g. total counts, location) from picking. This implementation does create a catch: `_onPickSubLayer` must be bound every frame for `this.state` to work.